lib/data/ddata.js

```diff
@@ -47,11 +51,11 @@ function init( ) {
     partition('treatments');
     partition('devicestatus');
 
-    result.first.sgvs = ddata.sgvs;
-    result.first.cals = ddata.cals;
+    result.first.sgvs = ddata.sgvs.filter(filterMax);
+    result.first.cals = ddata.cals.filter(filterMax);
```

Comment:
Probably shouldn't filter cals, there won't be many and if we don't have 1 we can't show any raw